external-crates/move/crates/move-analyzer/src/analysis/typing_analysis.rs

```diff
@@ -349,73 +390,12 @@ impl TypingAnalysisContext<'_> {
                 sp!(_, N::TypeName_::ModuleType(sp!(_, mod_ident), struct_name)),
                 _,
             ) => {
-                self.add_struct_field_use_def(mod_ident, &struct_name.value(), use_name, use_pos);
+                self.add_field_use_def(mod_ident, &struct_name.value(), None, use_name, use_pos);
             }
             _ => (),
         }
     }
 
-    fn add_field_use_def_internal(
```

Comment:
I had to pull `add_variant_field_use_def` and `add_struct_field_use_def` functions into a single one to make Rust borrow-checker happy (but also it reduced some code redundancy)